README.md

```diff
@@ -63,7 +92,33 @@ An example of three clusters configured to use with Submariner would look like t
 
 # Installation
 
-## Setup
+Submariner installation can be done manually using helm charts or by using its operator. Both methods are described here.
+
+## Installation using operator
+
+An operator is a tool to package, manage and deploy any Kubernetes application. A submariner operator will soon be available on [OperatorHub.io](https://operatorhub.io).
+
+### Building the operator
+
+```
+cd operators/go
+make build-operator
+```
+
+### Deploying Submariner using the Operator
+
+After generating the Operator (see docs above), your newly generated operator
+is automatically fully integrated into the Submariner CI automation. Simply use
+the `deploytool` flag to the standard `make` commands.
+
+```make ci e2e status=keep deploytool=operator```
+
+A large set of verifications for the Operator and the resulting Submariner
+deployment will automatically run during and after the deployment.
+
```

Comment:
As @skitt mentioned, this part does not belong here, this has to be user oriented, and we still have no user guide about how to deploy with the operator, this is just for development, we should probably move this to the other document where we talk about using make e2e

Reply:
I will add that part in a separate PR as mentioned above.